docs/reference/edot-collector/config/authentication-methods.md

```diff
@@ -0,0 +1,242 @@
+---
+navigation_title: Authentication methods
+description: Learn how to configure authentication for the EDOT Collector using API key authentication, bearer token authentication, and other security methods.
+applies_to:
+  stack: ga
+  serverless: ga
+  product:
+    edot_collector: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# Authentication methods for the EDOT Collector
+
+The {{edot}} Collector supports multiple authentication methods to secure connections and ensure only authorized clients can send telemetry data. This guide covers the available authentication extensions and how to configure them.
+
+## Overview
+
+Authentication in the EDOT Collector is handled through extensions that implement the `extensionauth` interface. These extensions can be used to:
+
+- Authenticate incoming requests from SDKs and other collectors.
+- Authenticate outgoing requests to external services.
+- Secure internal Collector communications.
```

Comment:
```
Secure internal Collector communications.
```

What does this exactly mean? I don't think these authentication methods secure anything internally within a single collector.